pdr_backend/aimodel/aimodel_plotdata.py

```diff
@@ -17,14 +16,18 @@ def __init__(
         ytrue_train: np.ndarray,
         colnames: List[str],
         slicing_x: np.ndarray,
+        sweep_vars: Optional[List[int]] = None,
```

Comment:
I don't think that's feasible. We create the aimodel itself, then we sweep through it. We don't even know what we are going to sweep when we crate it. I added sweep_vars as None when we don't sweep anything. We can adjust this as needed, but in case the property doesn't exist I consider all vars.